### DISCUSSION TITLE
How to display user mobile phone in pnp search result People Card

### DISCUSSION THREAD

**Discussion by vizsnyai** [microsoft-search/pnp-modern-search#3395] [Q&A] [answered]
Hi,

How to display user mobile phone in pnp search result People Card? 
What slot did you use or how can it be solved?

Thanks

**Comment by kasperbolarsen** [comment, marked as answer]
Hi
First of all, ensure that the user profiles actually have a value in the Mobile Phone field:
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/bf84d6f8-c5ec-4729-a705-9fde388c3239)

As I recall it, the Mobile Phone field is NOT automatically synced from AD/Entra so you might have to set up your own sync job. ( the internal name for Mobile Phone is CellPhone)

Once you have data in that field in the Search index ( I recommend using either SharePoint Query Tool or SP Editor to verify data) you should be able to use the managed property "MobilePhone"


![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/8f522792-fb11-42a4-838b-89c3b42498db)

Add "MobilePhone" to Selected properties in the Results web part, Go to "Layout slots"->Customize

add MobilePhone and WorkPhone
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/4e73ed11-f3e2-43bd-982b-ff8bf71adc5d)




 go to "Manage people fields" on page 2
Click Manage Prople fields
Change "optional text" to use Handlebars
insert {{slot item @root.slots.WorkPhone}} / {{slot item @root.slots.MobilePhone}}

And now the mobile number is displayed to the right of the Work Phone number :-)
![image](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/6a7cbf10-02c6-4c99-9465-62587e91be7b)

**Reply by vizsnyai** [reply]
Hi,

Thanks!
I managed to synchronize the mobile phone data from Entra.
The data is included in the CellPhone value.
![properties](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/61211581-d155-4b8c-989c-abd8996360cf)

I checked with the SharePoint Search Query Tool, the data comes back.
![tester](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/54d4c86a-b2fb-4175-b78a-90b289c700f6)

I added the MobilPhone value on the Slot layouts interface. I entered it manually in the Slot Field column because it was not possible to select it. Is that good?
![layouts](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/5767e9f1-c99c-499c-bdcb-2e0d6bc042b7)

I added Handlebar expression and then published the page.
![handlebar](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/34ba37e7-45b6-4b7c-918f-8beb610b0c71)

Unfortunately, the mobile number is still not displayed.
![results](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/b08caa46-52bc-4a79-b012-53e4be16dde5)

What am I doing wrong in the settings?
Thanks!

**Reply by kasperbolarsen** [reply]
Did you remember to add MobilePhone in Selected Properties ?

**Reply by vizsnyai** [reply]
Where can I do that?

**Reply by kasperbolarsen** [reply]
![169262224-22612485-1b4a-41c3-89be-baf86385102a.png](https://github.com/microsoft-search/pnp-modern-search/assets/20593570/54bfb552-3597-499b-aedb-68debcb6d61c)

**Reply by vizsnyai** [reply]
Then this could be the problem, because the MobilPhone property does not appear in the list.
Where can I add this property here?
![properties](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/5be29821-3883-4109-a93c-43970c643c09)

**Reply by kasperbolarsen** [reply]
You can just type type it in the field 🙂

We might in the future drop the checkbox before we can't ensure that all potential usable properties are shown.

**Reply by vizsnyai** [reply]
Mobile Phone is specified for managed properties
![MobilePhone](https://github.com/microsoft-search/pnp-modern-search/assets/74121481/4604f02b-8c0a-4918-9649-2f0e19f27a90)

**Reply by kasperbolarsen** [reply]
Great, and you have typed it in Selected Properties in the Results web part?

( is it Hungarian in the screen shorts?)

**Reply by vizsnyai** [reply]
Yes, it is hungarian.

Okay great.
It works :)
The solution was to add it manually to the selected properties, and then you could select it from the list.
Thank you very much for your help!

**Comment by kasperbolarsen** [comment]
You are very welcome :-)
